tests/data/function.py

```diff
@@ -230,7 +230,7 @@ def trailing_comma():
     }
 
 
-def f(a, **kwargs) -> A:
+def f(a, **kwargs,) -> A:
```

Comment:
Is this intended?  I don’t think human devs would write a one-line function signature like this.

Reply:
I don't see it as avoidable: we're now preserving trailing commas, and it's pretty hard to remove them only in function signatures as far as I can tell. I'm very open to suggestions, but as you say I don't think human devs would write a function like this so it's moot.

Reply:
Preserving seems ok!  Even if people have to point that out in code review if it happens, there is still a lot of time saved and arguments avoided thanks to all the other things that black changes.

Thanks for replying!